assets/js/consent-mode/consent-mode.js

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+const { isEqual } = require( 'lodash' );
```

Comment:
This [file is processed by Babel/Webpack](https://github.com/google/site-kit-wp/blob/36910fa94284f668e9fda888ce0c5ff7ac69f900/webpack/frontendModules.config.js#L39-L40), so is there any reason we can't use an ES6 `import` statement like in the rest of our code?

Not sure why we don't have ESLint rules enabled here as I'd think they'd catch this, or maybe we do and are just missing a rule for this! 😅 

```suggestion
import { isEqual } from 'lodash';
```